src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -748,12 +756,12 @@ static void create_rr_locked(grpc_exec_ctx* exec_ctx, glb_lb_policy* glb_policy,
       grpc_lb_policy_create(exec_ctx, "round_robin", args);
   if (new_rr_policy == NULL) {
     gpr_log(GPR_ERROR,
-            "Failure creating a RoundRobin policy for serverlist update with "
-            "%lu entries. The previous RR instance (%p), if any, will continue "
-            "to be used. Future updates from the LB will attempt to create new "
-            "instances.",
-            (unsigned long)glb_policy->serverlist->num_servers,
-            (void*)glb_policy->rr_policy);
+            "[grpclb %p] Failure creating a RoundRobin policy for serverlist "
+            "update with %lu entries. The previous RR instance (%p), if any, "
```

Comment:
Suggest using `PRIuPTR` here to avoid the cast.

Reply:
Done